pkg/issuer/acme/dns/util/dns.go

```diff
@@ -16,5 +16,8 @@ func DNS01Record(domain, value string) (string, string, int) {
 	if err == nil && r.Rcode == dns.RcodeSuccess {
 		fqdn = updateDomainWithCName(r, fqdn)
 	}
-	return fqdn, value, 60
```

Comment:
Can you open a separate PR with this patch? It seems valuable outside the context of this PR!

Reply:
Sure